src/components/Welcomes/QRScanner/index.tsx

```diff
@@ -17,7 +18,7 @@ export default function QRScanner(
     const { scanning, onResult, onError, ...videoProps } = props
     const video = useRef<HTMLVideoElement | null>(null)
 
-    useQRCodeScan(video, scanning, onResult, onError)
+    useQRCodeScan(video, scanning, x => compressBackupFile(decompressBackupFile(x)), onError)
```

Comment:
Feels like we are breaking some abstraction here. If you are ditching onResult, remove it completely.

Reply:
… and what is this meant to do exactly? To make sure that the key is a valid compressed point?

Reply:
Oh it's a mistake let me fix it.